modules/auxiliary/scanner/http/wordpress_contus_video_gallery_sqli.rb

```diff
@@ -0,0 +1,74 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'uri'
```

Comment:
Ah, cool. Let me address these this evening.

On Sun, Apr 19, 2015 at 11:25 PM, Christian Mehlmauer <
notifications@github.com> wrote:

> In modules/auxiliary/scanner/http/wordpress_contus_video_gallery_sqli.rb
> <https://github.com/rapid7/metasploit-framework/pull/5198#discussion_r28663465>
> :
>
> > @@ -0,0 +1,74 @@
> > +##
> > +# This module requires Metasploit: http://metasploit.com/download
> > +# Current source: https://github.com/rapid7/metasploit-framework
> > +##
> > +
> > +require 'uri'
>
> I think this is not needed
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rapid7/metasploit-framework/pull/5198/files#r28663465>
> .
>



-- 
http://volatile-minds.blogspot.com -- blog
http://www.volatileminds.net -- website